.env

```diff
@@ -2,3 +2,4 @@ COLL_8_BASE_URL=https://ship-api-uat.drop2shop.ie/api
 COLL_8_CLIENT_ID=test_id
 COLL_8_CLIENT_SECRET=test_secret
 COLL_8_ACCOUNT=test_account
+COLL_8_TOKEN=test_token
```

Comment:
This can be removed.

Reply:
It is needed for running the specs (so as not to hard code the token).